forage-android/src/main/java/com/joinforage/forage/android/collect/BTPinCollector.kt

```diff
@@ -25,6 +28,7 @@ internal class BTPinCollector(
                 "X-KEY" to encryptionKey,
                 "Merchant-Account" to merchantAccount,
                 "BT-PROXY-KEY" to PROXY_ID,
+                "IDEMPOTENCY-KEY" to UUID.randomUUID().toString(),
```

Comment:
is this related to the encryption key/alias/token changes or are we adding this to support idempotent balance checks in the android SDK?

Reply:
Turns out this was required the whole time 👀